iterable-extension/src/main/java/com/mparticle/ext/iterable/IterableExtensionLogger.java

```diff
@@ -35,10 +35,32 @@ public IterableExtensionLogger(String awsRequestId, BlobbyClient bc, boolean has
     this.hasBlobbyEnabled = hasBlobbyEnabled;
   }
 
-  public void logIterableApiError(retrofit2.Call<?> preparedCall,
-                                  Response<?> response, UUID mparticleEventId, Boolean isRetriable) {
-    Map<String, String> blobbyLogMessage = new HashMap<>();
+  public void logIterablePayloadForSpecificAPIKey (String apiKey, retrofit2.Call<?> preparedCall,
+                                                   Response<?> response, UUID mparticleEventId) {
+    if(apiKey.endsWith("6b583247")) {
+      Map<String, String> blobbyLogMessage = new HashMap<>();
+      String requestBody = getRequestBody(preparedCall);
+      String responseBody = getResponseBody(response);
+      String blobbyId = getBlobbyId(blobbyLogMessage, requestBody, responseBody);
+
+      String requestId = mparticleEventId != null ? mparticleEventId.toString() : "Error";
+      String url = response.raw().request().url().encodedPath();
+      String httpStatus = String.valueOf(response.code());
+
+      Map<String, String> logMessage = new HashMap<>();
+      logMessage.put("apiKey", apiKey);
+      logMessage.put("awsRequestId", awsRequestId);
+      logMessage.put("mParticleEventId", requestId);
+      logMessage.put("blobbyId", blobbyId);
+      logMessage.put("message", "Logging requests with API Key ending in 6b583247");
+      logMessage.put("url", url);
+      logMessage.put("httpStatus", httpStatus);
+      String messageJson = gson.toJson(logMessage);
+      System.out.println(messageJson);
```

Comment:
Is logging to stdout going to send the log line where we want it? Should we log to `logger.logMessage` or will that not work for this?

Reply:
Logging to stdout will write to Cloudwatch. That's how it currently works. Probably not the best, but fine since this is going away soon